src/sys/socket/mod.rs

```diff
@@ -810,45 +766,9 @@ pub fn accept(sockfd: RawFd) -> Result<RawFd> {
 ///
 /// [Further reading](http://man7.org/linux/man-pages/man2/accept.2.html)
 pub fn accept4(sockfd: RawFd, flags: SockFlag) -> Result<RawFd> {
```

Comment:
Looks like OSX lacks accept4, and NetBSD only has it on the unstable development version.  So you should gate this function.